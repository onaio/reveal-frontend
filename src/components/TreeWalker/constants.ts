export const ACTIVE = 'Active';
export const FEATURE = 'Feature';
export const FIND_BY_ID = 'location/findByJurisdictionIds';
export const FIND_BY_PROPERTIES = 'location/findByProperties';
export const LOCATION = 'location';
export const COULDNT_LOAD_CHILDREN = 'Could not load children';
export const COULDNT_LOAD_PARENTS = 'Could not load parents';
export const NOT_AVAILABLE = 'N/A';
