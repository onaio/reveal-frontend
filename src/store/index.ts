import { store } from '@opensrp/store';

export default store;
