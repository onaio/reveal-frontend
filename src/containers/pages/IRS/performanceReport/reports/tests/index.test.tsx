import reducerRegistry from '@onaio/redux-reducer-registry';
import { mount, shallow } from 'enzyme';
import flushPromises from 'flush-promises';
import { createBrowserHistory } from 'history';
import React from 'react';
import { act } from 'react-dom/test-utils';
import { Provider } from 'react-redux';
import { Router } from 'react-router';
import ConnectedIRSPerfomenceReport, { IRSPerfomenceReport } from '..';
import { PERFORMANCE_REPORT_IRS_PLAN_URL } from '../../../../../../constants';
import store from '../../../../../../store';
import PlansReducer, {
  genericFetchPlans,
  GenericPlan,
  reducerName as PlansReducerName,
} from '../../../../../../store/ducks/generic/plans';
import * as plans from '../../../../../../store/ducks/generic/tests/fixtures';
import DataCollectorreducer, {
  FetchIRSDataCollectors,
  reducerName as CollectorReducerName,
} from '../../../../../../store/ducks/opensrp/performanceReports/IRS/dataCollectorReport';
import DistrictReducer, {
  FetchIRSDistricts,
  reducerName as DistrictReducerName,
} from '../../../../../../store/ducks/opensrp/performanceReports/IRS/districtReport';
import SOPByDateReducer, {
  reducerName as SOPByDateReducerName,
} from '../../../../../../store/ducks/opensrp/performanceReports/IRS/sopByDateReport';
import SOPReducer, {
  FetchIRSSOPs,
  reducerName as SOPReducerName,
} from '../../../../../../store/ducks/opensrp/performanceReports/IRS/sopReport';
import * as fixtures from '../../../../../../store/ducks/opensrp/performanceReports/IRS/tests/fixtures';

jest.mock('../../../../../../configs/env');

const mock: any = jest.fn();
const history = createBrowserHistory();

const props = {
  history,
  location: mock,
  match: {
    isExact: true,
    params: {
      planId: fixtures.districtsReport[0].plan_id,
    },
    path: `${PERFORMANCE_REPORT_IRS_PLAN_URL}/:planId`,
    url: `${PERFORMANCE_REPORT_IRS_PLAN_URL}/${fixtures.districtsReport[0].plan_id}`,
  },
};

/** register the reducers */
reducerRegistry.register(DistrictReducerName, DistrictReducer);
reducerRegistry.register(CollectorReducerName, DataCollectorreducer);
reducerRegistry.register(SOPReducerName, SOPReducer);
reducerRegistry.register(SOPByDateReducerName, SOPByDateReducer);
reducerRegistry.register(PlansReducerName, PlansReducer);

plans.plans[0].plan_id = fixtures.districtsReport[0].plan_id;
store.dispatch(genericFetchPlans(plans.plans as GenericPlan[]));

describe('containers/pages/IRS/performanceReport/reports/', () => {
  beforeEach(() => {
    jest.resetAllMocks();
  });

  it('renders without crashing', () => {
    shallow(
      <Router history={history}>
        <IRSPerfomenceReport {...props} />
      </Router>
    );
  });

  it('renders correctly district page', async () => {
    const supersetServiceMock: any = jest.fn();
    supersetServiceMock.mockImplementationOnce(async () => fixtures.districtsReport);
    (props as any).service = supersetServiceMock;
    const wrapper = mount(
      <Provider store={store}>
        <Router history={history}>
          <ConnectedIRSPerfomenceReport {...props} />
        </Router>
      </Provider>
    );

    await act(async () => {
      await flushPromises();
    });
    wrapper.update();

    expect(wrapper.find('.page-title').text()).toEqual(
      'IRS Performance Reporting: IRS 2019-09-05 TEST'
    );
    wrapper.find('.breadcrumb li').forEach((list, index) => {
      expect(list.text()).toMatchSnapshot(`districts page breadcrumbs ${index + 1}`);
    });
    wrapper.find('.th').forEach((header, index) => {
      expect(header.text()).toMatchSnapshot(`districts table header ${index + 1}`);
    });
    wrapper.find('.tr .td').forEach((data, index) => {
      expect(data.text()).toMatchSnapshot(`districts table data ${index + 1}`);
    });
  });

  it('renders correctly data collectors page', async () => {
    const supersetServiceMock: any = jest.fn();
    store.dispatch(FetchIRSDistricts(fixtures.districtsReport));
    supersetServiceMock.mockImplementationOnce(async () => fixtures.dataCollectorsReport);
    supersetServiceMock.mockImplementationOnce(async () => fixtures.sopReport);
    supersetServiceMock.mockImplementationOnce(async () => fixtures.sopByDateReport);
    const jurisdictionId = fixtures.districtsReport[0].district_id;
    const planId = fixtures.districtsReport[0].plan_id;
    const newProps = {
      history,
      location: mock,
      match: {
        isExact: true,
        params: {
          jurisdictionId,
          planId,
        },
        path: `${PERFORMANCE_REPORT_IRS_PLAN_URL}/:planId/:jurisdictionId`,
        url: `${PERFORMANCE_REPORT_IRS_PLAN_URL}/${planId}/${jurisdictionId}`,
      },
      service: supersetServiceMock,
    };
    const wrapper = mount(
      <Provider store={store}>
        <Router history={history}>
          <ConnectedIRSPerfomenceReport {...newProps} />
        </Router>
      </Provider>
    );

    await act(async () => {
      await flushPromises();
    });
    wrapper.update();

    expect(wrapper.find('.page-title').text()).toEqual(
      'IRS Performance Reporting: Nchelenge District (2020)'
    );
    wrapper.find('.breadcrumb li').forEach((list, index) => {
      expect(list.text()).toMatchSnapshot(`data collector page breadcrumbs ${index + 1}`);
    });
    wrapper.find('.th').forEach((header, index) => {
      expect(header.text()).toMatchSnapshot(`data collector table header ${index + 1}`);
    });
    wrapper.find('.tr .td').forEach((data, index) => {
      expect(data.text()).toMatchSnapshot(`data collector table data ${index + 1}`);
    });
  });

  it('renders correctly spray operator page', async () => {
    const supersetServiceMock: any = jest.fn();
    store.dispatch(FetchIRSDistricts(fixtures.districtsReport));
    store.dispatch(FetchIRSDataCollectors(fixtures.dataCollectorsReport));
    supersetServiceMock.mockImplementationOnce(async () => fixtures.sopReport);
    supersetServiceMock.mockImplementationOnce(async () => fixtures.sopByDateReport);
    const jurisdictionId = fixtures.districtsReport[0].district_id;
    const planId = fixtures.districtsReport[0].plan_id;
    const dataCollector = fixtures.dataCollectorsReport[0].data_collector;
    const newProps = {
      history,
      location: mock,
      match: {
        isExact: true,
        params: {
          dataCollector,
          jurisdictionId,
          planId,
        },
        path: `${PERFORMANCE_REPORT_IRS_PLAN_URL}/:planId/:jurisdictionId/:dataCollector`,
        url: `${PERFORMANCE_REPORT_IRS_PLAN_URL}/${planId}/${jurisdictionId}/${dataCollector}`,
      },
      service: supersetServiceMock,
    };
    const wrapper = mount(
      <Provider store={store}>
        <Router history={history}>
          <ConnectedIRSPerfomenceReport {...newProps} />
        </Router>
      </Provider>
    );

    await act(async () => {
      await flushPromises();
    });
    wrapper.update();

    expect(wrapper.find('.page-title').text()).toEqual(
      'IRS Performance Reporting: NL6:Lillian Mulubwa'
    );
    wrapper.find('.breadcrumb li').forEach((list, index) => {
      expect(list.text()).toMatchSnapshot(`sop page breadcrumbs ${index + 1}`);
    });
    wrapper.find('.th').forEach((header, index) => {
      expect(header.text()).toMatchSnapshot(`sop table header ${index + 1}`);
    });
    wrapper.find('.tr .td').forEach((data, index) => {
      expect(data.text()).toMatchSnapshot(`sop table data ${index + 1}`);
    });
  });

  it('renders correctly spray operator by date page', async () => {
    const supersetServiceMock: any = jest.fn();
    store.dispatch(FetchIRSDistricts(fixtures.districtsReport));
    store.dispatch(FetchIRSDataCollectors(fixtures.dataCollectorsReport));
    store.dispatch(FetchIRSSOPs(fixtures.sopReport));
    supersetServiceMock.mockImplementationOnce(async () => fixtures.sopByDateReport);
    const jurisdictionId = fixtures.districtsReport[0].district_id;
    const planId = fixtures.districtsReport[0].plan_id;
    const dataCollector = fixtures.dataCollectorsReport[0].data_collector;
    const sop = fixtures.sopReport[0].sop;
    const newProps = {
      history,
      location: mock,
      match: {
        isExact: true,
        params: {
          dataCollector,
          jurisdictionId,
          planId,
          sop,
        },
        path: `${PERFORMANCE_REPORT_IRS_PLAN_URL}/:planId/:jurisdictionId/:dataCollector/:sop`,
        url: `${PERFORMANCE_REPORT_IRS_PLAN_URL}/${planId}/${jurisdictionId}/${dataCollector}/${sop}`,
      },
      service: supersetServiceMock,
    };
    const wrapper = mount(
      <Provider store={store}>
        <Router history={history}>
          <ConnectedIRSPerfomenceReport {...newProps} />
        </Router>
      </Provider>
    );

    await act(async () => {
      await flushPromises();
    });
    wrapper.update();

    expect(wrapper.find('.page-title').text()).toEqual(
      'IRS Performance Reporting: ZM-SOP-2019-21017:Chomba Gladys'
    );
    wrapper.find('.breadcrumb li').forEach((list, index) => {
      expect(list.text()).toMatchSnapshot(`sop by date page breadcrumbs ${index + 1}`);
    });
    wrapper.find('.th').forEach((header, index) => {
      expect(header.text()).toMatchSnapshot(`sop by date table header ${index + 1}`);
    });
    wrapper.find('.tr .td').forEach((data, index) => {
      expect(data.text()).toMatchSnapshot(`sop by date table data ${index + 1}`);
    });
  });
});
