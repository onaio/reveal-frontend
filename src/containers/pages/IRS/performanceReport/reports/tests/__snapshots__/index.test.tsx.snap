// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly data collectors page: data collector page breadcrumbs 1 1`] = `"Home"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly data collectors page: data collector page breadcrumbs 2 1`] = `"IRS Performance Reporting"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly data collectors page: data collector page breadcrumbs 3 1`] = `"IRS 2019-09-05 TEST"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly data collectors page: data collector page breadcrumbs 4 1`] = `"Nchelenge District (2020)"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly data collectors page: data collector table data 1 1`] = `"NL6:Lillian Mulubwa"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly data collectors page: data collector table data 2 1`] = `"0"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly data collectors page: data collector table data 3 1`] = `"1"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly data collectors page: data collector table data 4 1`] = `"13:55"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly data collectors page: data collector table data 5 1`] = `"14:01"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly data collectors page: data collector table data 6 1`] = `"00:05"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly data collectors page: data collector table data 7 1`] = `"0.34"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly data collectors page: data collector table data 8 1`] = `""`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly data collectors page: data collector table header 1 1`] = `"Data Collector"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly data collectors page: data collector table header 2 1`] = `"Number Days Worked"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly data collectors page: data collector table header 3 1`] = `"Average Structure Per Day"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly data collectors page: data collector table header 4 1`] = `"Average Start Time"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly data collectors page: data collector table header 5 1`] = `"Average End Time"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly data collectors page: data collector table header 6 1`] = `"Average Duration in the Field per Day"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly data collectors page: data collector table header 7 1`] = `"Insectcide Usage Rate*"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly data collectors page: data collector table header 8 1`] = `"Data Quality Check"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly district page: districts page breadcrumbs 1 1`] = `"Home"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly district page: districts page breadcrumbs 2 1`] = `"IRS Performance Reporting"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly district page: districts page breadcrumbs 3 1`] = `"IRS 2019-09-05 TEST"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly district page: districts table data 1 1`] = `"Nchelenge District (2020)"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly district page: districts table data 2 1`] = `"0"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly district page: districts table data 3 1`] = `"6"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly district page: districts table data 4 1`] = `"15:58"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly district page: districts table data 5 1`] = `"16:01"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly district page: districts table data 6 1`] = `"00:03"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly district page: districts table data 7 1`] = `"0.61"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly district page: districts table data 8 1`] = `""`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly district page: districts table header 1 1`] = `"District"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly district page: districts table header 2 1`] = `"Number of Days Teams in Field"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly district page: districts table header 3 1`] = `"Average Structure Per Day"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly district page: districts table header 4 1`] = `"Average Start Time"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly district page: districts table header 5 1`] = `"Average End Time"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly district page: districts table header 6 1`] = `"Average Duration in the Field per Day"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly district page: districts table header 7 1`] = `"Insectcide Usage Rate*"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly district page: districts table header 8 1`] = `"Data Quality Check"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator by date page: sop by date page breadcrumbs 1 1`] = `"Home"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator by date page: sop by date page breadcrumbs 2 1`] = `"IRS Performance Reporting"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator by date page: sop by date page breadcrumbs 3 1`] = `"IRS 2019-09-05 TEST"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator by date page: sop by date page breadcrumbs 4 1`] = `"Nchelenge District (2020)"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator by date page: sop by date page breadcrumbs 5 1`] = `"NL6:Lillian Mulubwa"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator by date page: sop by date page breadcrumbs 6 1`] = `"ZM-SOP-2019-21017:Chomba Gladys"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator by date page: sop by date table data 1 1`] = `"1"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator by date page: sop by date table data 2 1`] = `"2020-08-25"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator by date page: sop by date table data 3 1`] = `"1"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator by date page: sop by date table data 4 1`] = `"1"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator by date page: sop by date table data 5 1`] = `"0"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator by date page: sop by date table data 6 1`] = `"0"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator by date page: sop by date table data 7 1`] = `"0"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator by date page: sop by date table data 8 1`] = `"13:55"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator by date page: sop by date table data 9 1`] = `"14:01"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator by date page: sop by date table data 10 1`] = `"00:05"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator by date page: sop by date table data 11 1`] = `"2"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator by date page: sop by date table data 12 1`] = `""`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator by date page: sop by date table data 13 1`] = `"0"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator by date page: sop by date table data 14 1`] = `"0"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator by date page: sop by date table header 1 1`] = `" "`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator by date page: sop by date table header 2 1`] = `" "`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator by date page: sop by date table header 3 1`] = `" "`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator by date page: sop by date table header 4 1`] = `"Number of Structures"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator by date page: sop by date table header 5 1`] = `" "`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator by date page: sop by date table header 6 1`] = `" "`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator by date page: sop by date table header 7 1`] = `" "`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator by date page: sop by date table header 8 1`] = `" "`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator by date page: sop by date table header 9 1`] = `" "`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator by date page: sop by date table header 10 1`] = `" "`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator by date page: sop by date table header 11 1`] = `" "`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator by date page: sop by date table header 12 1`] = `" "`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator by date page: sop by date table header 13 1`] = `" "`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator by date page: sop by date table header 14 1`] = `"Number of Structures"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator by date page: sop by date table header 15 1`] = `"Not Sprayed"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator by date page: sop by date table header 16 1`] = `" "`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator by date page: sop by date table header 17 1`] = `" "`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator by date page: sop by date table header 18 1`] = `" "`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator by date page: sop by date table header 19 1`] = `" "`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator by date page: sop by date table header 20 1`] = `" "`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator by date page: sop by date table header 21 1`] = `" "`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator by date page: sop by date table header 22 1`] = `" "`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator by date page: sop by date table header 23 1`] = `"Day"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator by date page: sop by date table header 24 1`] = `"Date"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator by date page: sop by date table header 25 1`] = `"Found"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator by date page: sop by date table header 26 1`] = `"Sprayed"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator by date page: sop by date table header 27 1`] = `"Refused"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator by date page: sop by date table header 28 1`] = `"Other"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator by date page: sop by date table header 29 1`] = `"Total"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator by date page: sop by date table header 30 1`] = `"Average Start Time"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator by date page: sop by date table header 31 1`] = `"Average End Time"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator by date page: sop by date table header 32 1`] = `"Average Duration in the Field per Day"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator by date page: sop by date table header 33 1`] = `"Bottles/Sachets Used"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator by date page: sop by date table header 34 1`] = `"Data Quality Check"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator by date page: sop by date table header 35 1`] = `"Found Difference"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator by date page: sop by date table header 36 1`] = `"Sprayed Difference"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator page: sop page breadcrumbs 1 1`] = `"Home"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator page: sop page breadcrumbs 2 1`] = `"IRS Performance Reporting"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator page: sop page breadcrumbs 3 1`] = `"IRS 2019-09-05 TEST"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator page: sop page breadcrumbs 4 1`] = `"Nchelenge District (2020)"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator page: sop page breadcrumbs 5 1`] = `"NL6:Lillian Mulubwa"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator page: sop table data 1 1`] = `"ZM-SOP-2019-21017:Chomba Gladys"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator page: sop table data 2 1`] = `"0"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator page: sop table data 3 1`] = `"1"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator page: sop table data 4 1`] = `"1"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator page: sop table data 5 1`] = `"0"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator page: sop table data 6 1`] = `"1"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator page: sop table data 7 1`] = `"13:55"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator page: sop table data 8 1`] = `"14:01"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator page: sop table data 9 1`] = `"00:05"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator page: sop table data 10 1`] = `"0.34"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator page: sop table data 11 1`] = `""`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator page: sop table data 12 1`] = `"-5"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator page: sop table data 13 1`] = `"0"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator page: sop table header 1 1`] = `" "`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator page: sop table header 2 1`] = `" "`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator page: sop table header 3 1`] = `"Number of Structures(Latest submission)"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator page: sop table header 4 1`] = `" "`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator page: sop table header 5 1`] = `" "`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator page: sop table header 6 1`] = `" "`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator page: sop table header 7 1`] = `" "`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator page: sop table header 8 1`] = `" "`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator page: sop table header 9 1`] = `" "`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator page: sop table header 10 1`] = `" "`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator page: sop table header 11 1`] = `" "`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator page: sop table header 12 1`] = `"Spray Operator"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator page: sop table header 13 1`] = `"Number Days Worked"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator page: sop table header 14 1`] = `"Found"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator page: sop table header 15 1`] = `"Sprayed"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator page: sop table header 16 1`] = `"Not Sprayed"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator page: sop table header 17 1`] = `"Average Structure Per Day"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator page: sop table header 18 1`] = `"Average Start Time"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator page: sop table header 19 1`] = `"Average End Time"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator page: sop table header 20 1`] = `"Average Duration in the Field per Day"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator page: sop table header 21 1`] = `"Insectcide Usage Rate*"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator page: sop table header 22 1`] = `"Data Quality Check"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator page: sop table header 23 1`] = `"Found Difference (Latest Submission)"`;

exports[`containers/pages/IRS/performanceReport/reports/ renders correctly spray operator page: sop table header 24 1`] = `"Sprayed Difference (Latest Submission)"`;
